base/float.jl

```diff
@@ -844,8 +844,9 @@ end
 """
     floatmin(T = Float64)
 
-Return the smallest positive normal number representable by the floating-point
-type `T`.
+Return the smallest positive non-subnormal number representable by the floating-point type T.
+
+This is not the opposite of `floatmax`! See the Extended help section for more info.
```

Comment:
```suggestion
This is not the negation of `floatmax`. See the Extended help section for more info.
```

`typemin` and `typemax` are exactly or nearly negations of eachother for most signed types. `floatmin` and `floatmax` are opposites of a different nature, which can be described in detail in the extended help section.